### PR TITLE
initial scaffolding of the test tool app.

### PR DESCRIPTION
**Pull request by p0lyn0mial**
**What this PR does / why we need it**:
this PR introduces the initial scaffolding of the test tool app. At the moment it reads command line arguments and modifies the given manifest accordingly. 

**Which issue(s) this PR fixes** : see more: #47

### CODE REVIEW

File: test/tools/verify/cmd/main.go [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+		return "", err
+	}
+	content := fmt.Sprintf("%s", contentRaw)
+
+	for _, keyValuePair := range keyValuePairs {
+		kv := strings.Split(keyValuePair, "=")
+		if len(kv) != 2 {
+			continue
+		}
+		content = strings.Replace(content, kv[0], kv[1], -1)
```
Comment (mrIncompetent) [inline test/tools/verify/cmd/main.go]: Can we use golang templating here?

File: test/tools/verify/cmd/main.go [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+	contentRaw, err := ioutil.ReadFile(pathToManifest)
+	if err != nil {
+		return "", err
+	}
+	content := fmt.Sprintf("%s", contentRaw)
+
+	for _, keyValuePair := range keyValuePairs {
+		kv := strings.Split(keyValuePair, "=")
+		if len(kv) != 2 {
+			continue
```
Comment (p0lyn0mial) [inline test/tools/verify/cmd/main.go]: alternatively we could fail on error

File: test/tools/verify/cmd/main.go (hunk 1 above)
Comment (p0lyn0mial) [reply test/tools/verify/cmd/main.go]: I think we could. But doesn't that mean we would have to maintain a separate struct with predefined fields ? As the golang's template engine applies a template to the given data object (template.Execute). Given the fact that each manifest has slightly different set of dynamic fields replacing a substring in the given strings seems to be much easier. What do you think :) ?

Comment (mrIncompetent) [reply test/tools/verify/cmd/main.go]: Very good point. Using a simple string replace seems better!

File: test/tools/verify/cmd/main.go (hunk 2 above)
Comment (mrIncompetent) [reply test/tools/verify/cmd/main.go]: I would prefer failing early

Comment (p0lyn0mial) [reply test/tools/verify/cmd/main.go]: sounds good to me.

**Comment by p0lyn0mial** [comment]
@mrIncompetent comments addressed.

**Comment by p0lyn0mial** [comment]
yay!

**Pull request merged**